This is needed because under mock (especially with a systemd-coredump
handler on the system), no core files are created.

Submitted upstream: https://github.com/libunwind/libunwind/pull/418

diff --git a/tests/run-coredump-unwind b/tests/run-coredump-unwind
index 8d07742574602328..0c2b28c942477f7d 100755
--- a/tests/run-coredump-unwind
+++ b/tests/run-coredump-unwind
@@ -48,6 +48,10 @@ fi
     ./crasher backing_files
 ) 2>/dev/null
 COREFILE=$TEMPDIR/core*
+if ! test -f "$COREFILE"; then
+    echo "crasher process did not produce coredump, test skipped"
+    exit 77
+fi
 
 # magic option -testcase enables checking for the specific contents of the stack
 ./test-coredump-unwind $COREFILE -testcase `cat $TEMPDIR/backing_files`
